src/components/exportAndImport/importFromJson.ts

```diff
@@ -132,6 +132,7 @@ const getForeignKeyReplacements = (
           referenceTableName
         )
         if (referenceValue) foreignKeyReplacements[columnName] = referenceValue
+        else foreignKeyReplacements[columnName] = null
```

Comment:
<img width="951" alt="Screenshot 2022-11-15 at 5 46 52 PM" src="https://user-images.githubusercontent.com/5456533/201829452-a775a02f-1f84-4a68-a44f-eecd652a0340.png">

This mitigates this occasional error that I mentioned yesterday. The problem was (in this case), that the file record we were importing had a foreign key reference to application_serial and application_response, but these referenced records were not in the system we're importing into, and they're not part of the template snapshot, so it was throwing this error and not importing the file.

This change just means that instead of failing, it'll just put `null` into the fields it can't reconnect. Which makes sense, as we can't link to an application that doesn't exist (and we don't want to export all linked ones with a template export)!

Reply:
> I have added it to this issue: #852

Why? It's not longer an issue :)